.github/workflows/test.yml

```diff
@@ -11,33 +11,21 @@ jobs:
       matrix:
         os:
           - macos-latest
+          - ubuntu-latest
         ocaml-compiler:
-          - 5.0
-          - 4.14.1
-          - 4.13.1
-          - 4.08.1
-
+          - "5.0"
+          - "4.14"
+          - "4.13"
+          - "4.08"
+        exclude:
+          - os: macos-latest
+            ocaml-compiler: "4.08"
     runs-on: ${{ matrix.os }}
-
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-      - name: Use OCaml ${{ matrix.ocaml-compiler }}
-        uses: ocaml/setup-ocaml@v2
+      - uses: actions/checkout@v4
+      - uses: ocaml/setup-ocaml@v3
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
-      - name: Install dependencies
-        run: |
-          opam install . --deps-only --with-test
-
-      - name: Build
-        run: |
-          opam exec -- dune build
-
-      - name: Tests
-        run: |
-          opam exec -- dune build @runtest
-
-      - name: Opam Lint
-        run: |
-          opam lint uri.opam uri-re.opam uri-sexp.opam uri-bench.opam
+      - run: opam install . --deps-only --with-test
+      - run: opam exec -- dune build
+      - run: opam exec -- dune build @runtest
```

Comment:
@smorimoto could you also re-add the `lint` step that you have removed in your patch?

Reply:
Oops, sure!